src/components/view-pane/index.tsx

```diff
@@ -55,6 +58,15 @@ class ViewPaneBase extends React.PureComponent<ViewPaneProps, {}> {
 
   public render() {
     const {isQuerySpecific, plots} = this.props;
+    const {manualSpecificationOnly} = this.props.config;
+    const fieldCount = this.props.data.schema.fieldSchemas.length;
+
+    const relatedViews = !manualSpecificationOnly && fieldCount > 0 ? (
```

Comment:
I don't understand why do you have to check `fieldCount` here. 

Plus, why is it related to a PR for `manualSpecificationOnly`?